package.json

```
@@ -73,7 +73,8 @@
     "i18next-node-fs-backend": "^2.1.3",
     "is-online": "^9.0.1",
     "jquery": "^3.6.0",
-    "jquery-mousewheel": "^3.1.13"
+    "jquery-mousewheel": "^3.1.13",
+    "jsonschema": "^1.4.1"
```

We are now using fixed versions. This likely needs a rebase.
```suggestion
    "jsonschema": "1.4.1"
```